docs/basic-features/image-optimization.md

```diff
@@ -2,25 +2,15 @@
 description: Next.js supports built-in image optimization, as well as third party loaders for Imgix, Cloudinary, and more! Learn more here.
 ---
 
-# Image Component and Image Optimization
+# Optimizing Images
```

Comment:
Can we call this title `Image Optimization` which is how we talk about this feature everywhere else?